content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -16,79 +16,93 @@ further_reading:
 
 ## Overview
 
-Datadog Cloud Cost Management (CCM) automatically allocates costs of Kubernetes and ECS clusters to individual pods and tasks running in those clusters. Use cost metrics enriched with tags from pods, nodes, containers, and tasks to visualize container workload cost in the context of your entire cloud bill.
+Datadog Cloud Cost Management (CCM) automatically allocates AWS/Google costs of Kubernetes and AWS ECS clusters to individual pods and tasks running in those clusters. Use cost metrics enriched with tags from pods, nodes, containers, and tasks to visualize container workload cost in the context of your entire cloud bill.
 
 {{< img src="cloud_cost/container_cost_allocation/cost_allocation_table.png" alt="Cloud cost allocation table showing requests and idle costs over the past week" style="width:100%;" >}}
 
-For Kubernetes clusters, CCM allocates costs of EC2 instances and non-local EBS volumes. For ECS clusters, CCM allocates costs of EC2 instances.
+For Kubernetes clusters, CCM allocates costs of AWS/Google host instances and non-local AWS EBS volumes. For AWS ECS clusters, CCM allocates costs of AWS EC2 instances.
 
 ## Prerequisites
 
-1. Set up and configure [AWS Cloud Cost integration][1].
-2. For Kubernetes support, install the [**Datadog Agent**][2] in a Kubernetes environment using EC2 instances.
+1. Set up and configure the [Cloud Cost integration][1].
+2. For Kubernetes support, install the [**Datadog Agent**][2] in a Kubernetes environment.
      - Ensure that you enable the [**Orchestrator Explorer**][6] in your Agent configuration.
      - Container cost allocation requires **Agent version >= 7.27.0** and **Cluster Agent version >= 1.11.0**.
-     - Persistent Volume allocation requires **Agent version >= 7.46.0**
-3. For ECS support, set up [**Datadog Container Monitoring**][3] in ECS tasks.
+     - AWS Persistent Volume allocation requires **Agent version >= 7.46.0**
+     - Google supports agentless Kubernetes cost tracking with some limitations, [see below for more details](#agentless-google-kubernetes-costs)
```

Comment:
imo we don't need to mention this here in prereqs, the section existing below is enough.

Reply:
```suggestion
```